### DISCUSSION TITLE
validateReceiptIos returning {status:21002} in the response

### DISCUSSION THREAD

**Discussion by Shri0017** [hyochan/react-native-iap#2489] [General]
const receiptBody = {
              'receipt-data': purchase.transactionReceipt,
              password: "*****', 
            };
            const appleReceiptResponse = await validateReceiptIos(receiptBody,false);

I'm following same structure which is defined in doc.

**Comment by textual** [comment]
my understanding is that this is not supported on app anymore. i have facing similar issue in wanting to read the contents of the purchase transaction receipt. im sending to my server but still facing issues so came here to validate my approach. If this package can decode the returned purchase transaction receipt, then i can just send that back to my server. but this section indicates its not possible anymroe

https://react-native-iap.dooboolab.com/docs/guides/receipts#local-validation
